catalyst_voices/lib/pages/spaces/spaces_shell_page.dart

```diff
@@ -6,17 +6,19 @@ import 'package:flutter/material.dart';
 class SpacesShellPage extends StatelessWidget {
   final Space space;
   final Widget child;
+  final bool showSearch;
 
   const SpacesShellPage({
     super.key,
     required this.space,
     required this.child,
+    this.showSearch = false,
   });
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: VoicesAppBar(),
+      appBar: VoicesAppBar(showSearch: showSearch),
```

Comment:
I think it should be just disabled. It does not work for now. When we're ready to implement it should be visible in every space